content/en/docs/concepts/policy/pod-security-policy.md

```diff
@@ -413,20 +413,28 @@ minimum value of the first range as the default. Validates against all ranges.
 to be used by hostPath volumes. An empty list means there is no restriction on
 host paths used. This is defined as a list of objects with a single `pathPrefix`
 field, which allows hostPath volumes to mount a path that begins with an
-allowed prefix. For example:
+allowed prefix, and a `readOnly` field indicating it must be mounted read-only.
+For example:
 
 ```yaml
 allowedHostPaths:
   # This allows "/foo", "/foo/", "/foo/bar" etc., but
   # disallows "/fool", "/etc/foo" etc.
   # "/foo/../" is never valid.
   - pathPrefix: "/foo"
+    readOnly: true # only allow read-only mounts
 ```
 
 _Note: There are many ways a container with unrestricted access to the host
 filesystem can escalate privileges, including reading data from other
 containers, and abusing the credentials of system services, such as Kubelet._
 
+{{< warning >}}**WARNING:** writeable hostPath directory volumes allow containers to write 
```

Comment:
A stacked note and warning together is not a great docs practice. Can we rework this so that there don't need to be two back-to-back admonitions? If the warning must be kept, s/WARNING/Warning

Reply:
they both are warning worthy. I'll combine them into a single warning block.